cadl-tests/src/main/java/cadl/testserver/servicedriven2/DPGClient.java

```diff
@@ -167,8 +167,8 @@ public Response<BinaryData> putRequiredOptionalWithResponse(String requiredParam
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<BinaryData> postParametersWithResponse(
-            String contenttype, BinaryData parameter, RequestOptions requestOptions) {
-        return this.client.postParametersWithResponse(contenttype, parameter, requestOptions).block();
+            String contentTypePath, BinaryData parameter, RequestOptions requestOptions) {
+        return this.client.postParametersWithResponse(contentTypePath, parameter, requestOptions).block();
```

Comment:
seems cadl-ranch updated?

Reply:
Yes, it's discussed [here](https://github.com/Azure/cadl-ranch/pull/50#discussion_r948583808).

Reply:
ok, if they've released new version, we should be able to enable the models in cadl-ranch.

commit added.